filebeat/fileset/flags.go

```diff
@@ -0,0 +1,59 @@
+package fileset
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+// Modules related command line flags.
+var (
+	modulesFlag     = flag.String("module", "", "List of enabled modules (comma separated)")
```

Comment:
I had "modules" in the first prototype, then thought that "module" would be nicer for the most common case. But you're right, I think "modules" makes more sense overall.